applications/AdjointFluidApplication/tests/test_MainKratos.py

```diff
@@ -32,6 +32,12 @@ def __init__(self, ProjectParameters):
 
```

Comment:
ok, I'm fine with an alternative approach. I implemented these tests based on what I saw in other parts of kratos.